appscale/tools/remote_helper.py

```diff
@@ -1042,3 +1042,16 @@ def collect_appcontroller_crashlog(cls, host, keyname, is_verbose):
       message = "AppController at {0} crashed for reasons unknown.".format(host)
 
     return message
+
+  @classmethod
+  def get_command_output_from_remote(cls, host, command, keyname, user='root', shell=False):
```

Comment:
I had a problem using the existing ssh method while trying to call the command git ls remote to fetch all the tags, so I created this new method.